### PR TITLE
Branch UI

### PR DESCRIPTION
**Pull request by Jacky142857**
Changed ui to horizontal, and removed some fields.

### CODE REVIEW

**Review by janald99 — APPROVED** [review]
LGTM.

File: src/main/java/seedu/address/logic/commands/AppointmentCommand.java [hunk 1]
```diff
@@ -51,8 +51,7 @@ public CommandResult execute(Model model) throws CommandException {
 
         Client clientToEdit = lastShownList.get(index.getZeroBased());
         Client editedClient = new Client(
-                clientToEdit.getName(), clientToEdit.getDescription(), clientToEdit.getPhone(), clientToEdit.getEmail(),
-                clientToEdit.getAddress(), clientToEdit.getRemark(), appointment, clientToEdit.getTags()
+                clientToEdit.getName(), clientToEdit.getPhone(), appointment, clientToEdit.getTags()
```
Comment (WJunHong) [inline src/main/java/seedu/address/logic/commands/AppointmentCommand.java:54]: is it possible to have an appointment be in the same format?
E.g.  `clientToEdit.getAppointment()`

**Review by WJunHong — COMMENTED** [review]
Thanks for helping remove the unnecessary fields for clients

**Comment by WJunHong** [comment]
Do not merge this until next week!

**Pull request merged**